enhancements/machine-api/machine-health-checking.md

```diff
@@ -102,18 +102,21 @@ For a node notFound or a failed machine, the machine is considerable unrecoverab
 - Enable watching a pool of machines (based on a label selector).
 - Enable defining an unhealthy node criteria (based on a list of node conditions).
 - Enable setting a threshold of unhealthy nodes. If the current number is at or above this threshold no further remediation will take place. This can be expressed as an int or as a percentage of the total targets in the pool.
+- Enable pausing of remediation
 
 E.g:
 - I want my worker machines to be remediated when the backed node has `ready=false` or `ready=Unknown` condition for more than 10m.
 - I want remediation to temporary short-circuit if the 40% or more of the targets of this pool are unhealthy at the same time.
-
+- I want no remediation to happen while my cluster is upgrading its machines / nodes.
 
 ```yaml
 apiVersion: machine.openshift.io/v1beta1
 kind: MachineHealthCheck
 metadata:
   name: example
   namespace: openshift-machine-api
+  annotations:
+    cluster.x-k8s.io/paused": "clusterUpgrading"
```

Comment:
Not sure about what values are good for the annotation. It's not being checked, but we might want to utilize it to sync between multiple parties who want to set this annotation?